explore/src/main/scala/explore/proposal/ProposalTabContents.scala

```diff
@@ -229,7 +246,10 @@ object ProposalTabContents:
     )
     .useState(none[String])        // Submission error message
     .useStateView(none[Timestamp]) // CFP/Proposal Deadline
-    .render { (props, ctx, isUpdatingStatus, readonly, errorMsg, deadline) =>
+    .useStreamOnMount(
```

Comment:
This causes the whole tab contents to be recomputed and rerendered every second. Maybe it's better to move the timer to its own component?